examples/all-clusters-app/all-clusters-common/all-clusters-app.matter

```diff
@@ -1823,7 +1823,7 @@ server cluster NetworkCommissioning = 49 {
   }
 
   attribute(readonly) int8u maxNetworks = 0;
-  attribute(readonly) NetworkInfo networks[] = 1;
+  attribute(readonly) NetworkInfo networks[12] = 1;
```

Comment:
Where does the 12 come from?

Reply:
Umm ... ember ?!? ... probably somewhere in a zap file. This is the reason I am adding this here, to make the "code" (in this case 10K lines of json) reviewable. Will try to find the location.

Reply:
I tried to find the location ... and failed. If one exists, we can update code later. For now this size will go away soon (forgot to remove it from my current patch version)

Reply:
The 12 is coming straight from the XML:
```
<attribute side="server" code="0x0001" define="NETWORKS" type="ARRAY" entryType="NetworkInfo" length="12" writable="false" optional="false">Networks</attribute>
```

Good thing we ignore that `length` value.... ;)